test/resource_embedding_test.dart

```diff
@@ -35,8 +35,21 @@ void main() {
         .execute();
     expect(res.data[0]['messages'].length, 2);
     expect(res.data[1]['messages'].length, 0);
-    expect(res.data[2]['messages'].length, 0);
-    expect(res.data[3]['messages'].length, 0);
+    expect(res.data[0]['messages'][0]['id'], 2);
+  });
+
+  test('embedded order on multiple columns', () async {
```

Comment:
Not very critical, but just curious.  Why did you slightly change it from the one in supabase-js?

Reply:
I actually explained it in the first post. Was it unclear?

> I ignored the equivalent tests in postgres-js because they don't make sure that sort results are as expected.
> 
> https://github.com/supabase/postgrest-js/blob/e5f5dfae7da22f4ff63c11c124d65d0699b7b890/test/transforms.ts#L10-L17
> https://github.com/supabase/postgrest-js/blob/e5f5dfae7da22f4ff63c11c124d65d0699b7b890/test/resource-embedding.ts#L41-L48
> 
> The messages table has two records with the same username ("supabot"), so sorting by the column does not mean the expected behaviour is correctly tested. I think these should be fixed.

I mean, sorting by `username` in the `messages` table does not affect the result as the two records have the same user name. I think this one and the other one in supabase-js are inappropriate and should be fixed. Therefore I made the new test in this package more meaningful and robust by using two columns other than `username` and also checking the result of each of the two sortings.

Reply:
Sorry, don't know why I missed that. Thank you for implementing a better test! It's awesome!